nautobot/core/settings.py

```diff
@@ -557,6 +512,20 @@
     "django_prometheus.middleware.PrometheusAfterMiddleware",
 ]
 
+
+LOGGING = {"loggers": {"nautobot": {}}}
+setup_structlog_logging(
+    LOGGING,
+    INSTALLED_APPS,
+    MIDDLEWARE,
```

Comment:
Do these work as intended when adding additional apps via `EXTRA_INSTALLED_APPS` (https://docs.nautobot.com/projects/core/en/stable/user-guide/administration/configuration/optional-settings/#extra-applications) and/or when a Nautobot App adds its own middleware? (https://docs.nautobot.com/projects/core/en/stable/development/apps/api/nautobot-app-config/?h=middleware#optional-nautobotappconfig-attributes)